test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueListenerTests.cs

```diff
@@ -86,99 +83,65 @@ public QueueListenerTests(TestFixture fixture)
             messageFromCloud = await queue.GetMessageAsync();
             Assert.Equal(2, messageFromCloud.DequeueCount);
 
-            processor.Original = messageFromCloud;
-
             await listener.ProcessMessageAsync(new StorageQueueMessage(messageFromCloud), TimeSpan.FromMinutes(10), CancellationToken.None);
-        }
 
-        private class TestQueueProcessor : QueueProcessor
-        {
-            private CloudQueue _queue;
-            private CloudQueue _poisonQueue;
-            private CloudQueueMessage _original;
+            // Make sure the message was processed and deleted.
```

Comment:
Yes, for this test, that worked before as well. I just wanted a better validation so added it. For the other test (the renewal) -- it affects people using 7.x. We should try to get a release out as soon as this is in.